src-tauri/src/app_config.rs

```diff
@@ -47,6 +51,8 @@ impl Default for AppConfigFromFile {
             monero_address: default_monero_address(),
             gpu_mining_enabled: true,
             cpu_mining_enabled: true,
+            has_system_language_been_proposed: false,
```

Comment:
```suggestion
            use_system_default_locale: false,
```
Perhaps it's more intuitive to name it this?